website/styles/globals.css

```diff
@@ -8,7 +8,9 @@ body,
   @apply h-full;
   @apply antialiased;
 }
-
+.prose-docs > img {
```

Comment:
This would be better as a tailwind rule as below

Reply:
i forgot i added that.... trying to get anything to work